code/modules/vending/jobclothing.dm

```diff
@@ -242,3 +242,29 @@ ABSTRACT_TYPE(/obj/machinery/vending/jobclothing)
 #ifdef SEASON_WINTER
 		product_list += new/datum/data/vending_product(/obj/item/clothing/suit/puffer/sci, 2)
 #endif
+
+/obj/machinery/vending/jobclothing/syndicate
+	name = "Syndicate Apparel"
+	desc = "A vending machine that vends Syndicate clothing."
+	icon_state = "syndieclothing"
+	icon_panel = "snack-panel"
+	icon_off = "syndieclothing-off"
+	icon_broken = "syndieclothing-broken"
+	icon_fallen = "syndieclothing-fallen"
+	pay = 1
+	acceptcard = 1
+	req_access = list()
+
+	create_products(restocked)
+		..()
+		product_list += new/datum/data/vending_product(/obj/item/clothing/under/misc/syndicate, 4)
+		product_list += new/datum/data/vending_product(/obj/item/clothing/head/helmet/space/syndicate, 2)
+		product_list += new/datum/data/vending_product(/obj/item/clothing/suit/space/syndicate, 2)
+		product_list += new/datum/data/vending_product(/obj/item/tank/jetpack/syndicate, 2)
+		product_list += new/datum/data/vending_product(/obj/item/clothing/mask/gas/swat, 2, cost=PAY_IMPORTANT)
+		product_list += new/datum/data/vending_product(/obj/item/clothing/shoes/swat, 4)
+		product_list += new/datum/data/vending_product(/obj/item/storage/backpack/syndie, 1)
+		product_list += new/datum/data/vending_product(/obj/item/storage/backpack/satchel/syndie, 1)
+		product_list += new/datum/data/vending_product(/obj/item/storage/fanny/syndie, 1)
+
+		product_list += new/datum/data/vending_product(/obj/item/clothing/head/helmet/space/syndicate/commissar_cap, 1, hidden=1)
```

Comment:
```suggestion
```
Oh uh, I just noticed this was added - this is a nukie/podwars exclusive item that shouldn't appear in normal rounds. The stats on it are kind of crazy and it just doesn't fit outside of special round types.

Reply:
The stats on it are the same as the space helm except for 10% explosion resist but I can remove it 👍

Reply:
![image](https://github.com/goonstation/goonstation/assets/159719201/c730040b-cb50-4523-9903-a955eba7d82f)
Was meant as a fun syndicate headpiece so its fine to remove